pkg/download/version_zip.go

```diff
@@ -25,17 +24,18 @@ func VersionZipHandler(dp Protocol, lggr *log.Logger, eng *render.Engine) buffal
 		if err != nil {
 			err := errors.E(op, errors.M(mod), errors.V(ver), err)
 			lggr.SystemErr(err)
-			return c.Render(http.StatusInternalServerError, nil)
+			return c.Render(errors.Kind(err), nil)
 		}
+		defer verInfo.Zip.Close()
 
-		status := http.StatusOK
+		// Calling c.Response().Write will write the header directly
+		// and we would get a 0 status in the buffalo logs.
+		c.Render(200, nil)
 		_, err = io.Copy(c.Response(), verInfo.Zip)
 		if err != nil {
-			status = http.StatusInternalServerError
 			lggr.SystemErr(errors.E(op, errors.M(mod), errors.V(ver), err))
 		}
 
-		c.Response().WriteHeader(status)
```

Comment:
is it cool with you to not send an error back down to the client in this case? and are we just unable to do it b/c of the buffalo bug?

Reply:
@arschles Yes. The only possible error is that if the connection dropped or we went some falsy data, which in both cases the client will figure out. Plus, per the comment I put above, it's really our only option :)